benchmarks/linear_algebra/blas/level2/ger/ger.cpp

```diff
@@ -0,0 +1,74 @@
+/* 
```

Comment:
The name of this file does not respect the naming convention so that the code can be run using the compile_and_run.sh script. Please rename it to ger_generator.cpp

Reply:
done.